src/main/resources/logback.xml

```diff
@@ -0,0 +1,52 @@
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="AllAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/all.log</file>
+        <append>false</append>
+        <immediateFlush>true</immediateFlush>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="ModelAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/model.log</file>
+        <append>true</append>
+        <immediateFlush>true</immediateFlush>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="EvaluationAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/evaluator.log</file>
+        <append>true</append>
+        <immediateFlush>true</immediateFlush>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="INFO">
+        <appender-ref ref="STDOUT"/>
+        <appender-ref ref="AllAppender"/>
+    </root>
+
+    <logger name="org" level="INFO"/>
+    <logger name="io.netty" level="INFO"/>
+
+    <logger name="edu.twitter.model" level="INFO">
```

Comment:
I believe u don't need to put effective level if root is info

Reply:
w kman same :"D

Reply:
But if you mean it's redundant, I believe you are right. It will get its level automatically from the root logger if you don't explicitly assign one to it. I will change that 👍

Reply:
redundant

Reply:
Done 👍